tests/conftest.py

```diff
@@ -151,6 +151,20 @@ async def db():
     await conn.close()
 
 
+@pytest_asyncio.fixture
+async def insert_fake_resource():
+    async def _insert_fake_resource(database):
+        await database.execute(
+            """
+            INSERT INTO catalog (dataset_id, resource_id, url, priority, deleted)
+            VALUES ('61fd30cb29ea95c7bc0e1211', 'f8fb4c7b-3fc6-4448-b34f-81a9991f18ec', 'http://dev.local/', True, False)
+            ON CONFLICT (resource_id) DO NOTHING;
+            """
+        )
+
+    return _insert_fake_resource
```

Comment:
I like this, would there be a way to provide the dataset and resource IDs someway, so it's not sprinkled and hardcoded everywhere in the tests? Maybe `return _insert_fake_resource, dataset_id, resource_id`?

Or if this makes the fixture usage to annoying in tests:
```python
inserter, dataset_id, resource_id = insert_fake_resource
await inserter()
```
then have the `DATASET_ID` and `RESOURCE_ID` importable from a file, making them importable in the tests?

Reply:
I believe this fixture will be use in more tests in the future? I can see only one usage for now.
If not, maybe it would make more sense to be integrated in the test itself, making my previous comment irrelevant?

Reply:
Thought about the exact same thing in another PR :) fixing it here as well, thanks